### PR TITLE
docs: 3.8 release notes

### PR DESCRIPTION
**Pull request by vladar**
## Description

Release notes for v3.8. Please add anything you would like to mention from this release.

[Rendered version](https://github.com/gatsbyjs/gatsby/blob/release-notes-3.8/docs/docs/reference/release-notes/v3.8/index.md)

### CODE REVIEW

File: docs/docs/reference/release-notes/v3.8/index.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+---
+
+## Web Vitals Tracking
+
+By using the [`web-vitals`](https://github.com/GoogleChrome/web-vitals) package we've added support for tracking Web Vitals to `gatsby-plugin-google-analytics` & `gatsby-plugin-google-tagmanager` ([PR #31665](https://github.com/gatsbyjs/gatsby/pull/31665)). They send three metrics:
+
+- **Largest Contentful Paint (LCP)**: measures loading performance. To provide a good user experience, LCP should occur within 2.5 seconds of when the page first starts loading.
+- **First Input Delay (FID)**: measures interactivity. To provide a good user experience, pages should have a FID of 100 milliseconds or less.
+- **Cumulative Layout Shift (CLS)**: measures visual stability. To provide a good user experience, pages should maintain a CLS of 1 or less.
```
Comment (calcsam) [inline docs/docs/reference/release-notes/v3.8/index.md:33]:
```suggestion
- **Cumulative Layout Shift (CLS)**: measures visual stability. To provide a good user experience, pages should maintain a CLS of 0.1 or less.
```

Comment (calcsam) [reply docs/docs/reference/release-notes/v3.8/index.md:33]: https://web.dev/cls/

Comment (wardpeet) [reply docs/docs/reference/release-notes/v3.8/index.md]: @calcsam that's not true, in analytics it needs to be integered so that's why it's 1 here

**Review by KyleAMathews — APPROVED** [review]
:shipit:

**Pull request merged**